MatterControlLib/SetupWizard/DialogPage.cs

```diff
@@ -40,7 +40,7 @@ namespace MatterHackers.MatterControl
 	public class DialogPage : FlowLayoutWidget
 	{
 		protected GuiWidget headerRow;
-		protected FlowLayoutWidget contentRow;
+		public FlowLayoutWidget ContentRow { get; }
```

Comment:
I think we should undo this change. There are 42 types that derive from DialogPage and not a single one of them need this behavior. We can achieve the same result by adding a public ContentRow property to PrinterSetupWizard that acts as a proxy to this internal member, and not change the base type to support specialized behavior of a single new type